test/components/DayPickerRangeController_spec.jsx

```diff
@@ -1652,10 +1779,8 @@ describe('DayPickerRangeController', () => {
             focusedInput={START_DATE}
           />
         ));
-        // The first day click sets preventFocusChange to true, but it doesn't take effect until the
-        // second day click because onFocusChange is called before onDatesChange
         wrapper.instance().onDayClick(clickDate);
-        expect(focusedInput).to.equal(END_DATE);
```

Comment:
this does seem like a possible breaking change to me. @majapw?

Reply:
Yep! I'm gonna release it as v19